Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -441,38 +428,12 @@ class Gx2Fitter {
           TrackStatePropMask mask =
               ~(TrackStatePropMask::Smoothed | TrackStatePropMask::Filtered);
 
-          // Checks if an existing surface is found during the gx2f-iteration.
```

Comment:
This is removed now because it does not happen anymore?

Reply:
Yes, this whole block is the re-visiting-logic for the track states. But since we are always creating new ones in an empty container, we don't need them any more.